### ISSUE TITLE
[Feature Request] Add toggle to automatically import transaction from notification

### ISSUE THREAD

**Issue by NicolasAssouline** [dreautall/waterfly-iii#486]
Hi!

Long-time user of the app here, Waterfly-iii makes budgeting a lot easier :)

I noticed I rarely (if ever) need to edit anything when adding transactions through the notification the notification listener generates.
Constantly having to click on the notifications just to save what the notification listener already pre-filled starts getting tedious quickly, especially when making multiple transactions in a short span of time (and having to save each one individually).

Is it possible to add a toggle to auto-import transactions (and skipping the "Create transaction?" notification or maybe changing it into "Created transaction" so that users know it has been successfully imported).

The toggle would probably need to be configurable on a per-app basis (i.e skip the confirmation for my banking app, but still ask me for Google pay) since users would probably still want to manually review transactions for some apps.

Thanks for making such a great app!

**Comment by NicolasAssouline** [comment]
Yeah, I usually don't set the destination account (although I really should start doing that)
Thanks for adding this :)

**Comment by ballinger** [comment]
> ... banking app, but still ask me for Google pay)

How did you manage to add GPay notifications. I don't see it in the list.

**Comment by NicolasAssouline** [comment]
Unfortunately I only used it as an example, I can confirm I also can't see it in the notification list
